src/jobs/index.js

```diff
@@ -93,6 +99,41 @@ class Jobs {
     });
   }
 
+  /**
+   * Get stats from 'Já Não Dá Para Abastecer' website
+   */
+  static fuelStats() {
+    const rule = new schedule.RecurrenceRule();
+
+    rule.hour = new schedule.Range(0, 23, 1);
+    rule.minute = 0;
+    rule.second = 0;
+
+    schedule.scheduleJob(rule, async () => {
+      const stats = await Fuel.getFuelStats();
+
+      const actualTime = moment().format('H:mm');
+
+      const strHeader = `Estado às ${actualTime}`;
+
+      const strStatsTotal = `Total: ${stats.stations_total}`;
+
+      const strGeneral = 'Visão geral';
+      const strStatsNone = ` - Sem qualquer tipo de combustível: ${stats.stations_none}`;
+      const strStatsPartial = ` - Com algum tipo de combustível: ${stats.stations_partial}`;
+      const strStatsAll = ` - Com todos os tipos de combustível: ${stats.stations_all}`;
+
+      const strFaultByType = 'Faltas por tipo de combustível';
+      const strNoGasoline = ` - Sem gasolina: ${stats.stations_no_gasoline}`;
+      const strNoDiesel = ` - Sem gasóleo: ${stats.stations_no_diesel}`;
+      const strNoLpg = ` - Sem GPL: ${stats.stations_no_lpg}`;
+
+      const strMessage = `${strHeader}\n${strStatsTotal}\n\n${strGeneral}\n${strStatsNone}\n${strStatsPartial}\n${strStatsAll}\n${strFaultByType}\n${strNoGasoline}\n${strNoDiesel}\n${strNoLpg}`;
```

Comment:
This could be defined in an array of messages, and then concatenating all of them.

```js
const {
  stations_all,
  stations_none,
  stations_no_diesel,
  stations_no_gasoline,
  stations_no_lpg,
  stations_partial,
  stations_total,
} = await Fuel.getFuelStats();

const actualTime = moment().format('H:mm');

const messages = [
  `Estado às ${actualTime}`,
  `Total: ${stations_total}`,
  'Visão geral',
  ` - Sem qualquer tipo de combustível: ${stations_none}`,
  ` - Com algum tipo de combustível: ${stations_partial}`,
  ` - Com todos os tipos de combustível: ${stations_all}`,
  'Faltas por tipo de combustível',
  ` - Sem gasolina: ${stations_no_gasoline}`,
  ` - Sem gasóleo: ${stations_no_diesel}`,
  ` - Sem GPL: ${stations_no_lpg}`,
]

const message = messages.join('\n');
```

This approach eases message management. It also avoid having variables declared which we don't use anywhere else.

What you think?

Reply:
I agree with you, the code is more legible and simplified, will change it and make a new commit. Thanks for the feedback!